ci/cron/daily-compat.yml

```diff
@@ -391,6 +391,8 @@ jobs:
       - bash: |
           set -euo pipefail
           eval "$(dev-env/bin/dade-assist)"
+          
+          export SCAN_CLI_OPTS="-Xms1g -Xmx10g"
```

Comment:
shouldn’t we set this for both blackduck steps? You only set it for the second one.

Reply:
probably but the first haskell step seems to be fine so i just targeted it for the problematic one -- no harm in keeping it consistent for both though if this works